pkg/apiserver/registry/inventory/rest.go

```diff
@@ -176,9 +179,14 @@ func (r *REST) ConvertToTable(ctx context.Context, obj runtime.Object, tableOpti
 	}
 	var err error
 	table.Rows, err = metatable.MetaToTableRow(obj,
-		func(obj runtime.Object, m metav1.Object, name, age string) ([]interface{}, error) {
+		func(obj runtime.Object, _ metav1.Object, _, _ string) ([]interface{}, error) {
 			vpc := obj.(*runtimev1alpha1.Vpc)
 			return []interface{}{vpc.Name, vpc.Info.CloudProvider, vpc.Info.Region}, nil
 		})
 	return table, err
 }
+
+func (r *REST) Watch(ctx context.Context, options *internalversion.ListOptions) (watch.Interface, error) {
+	key, label, field := storage.GetSelectors(options)
```

Comment:
what are the filters you are supporting in watch, like namespace ??

Reply:
I think for now maybe only labelselector, I think if we want maybe fieldselector it will be good, but seems like namespace is not available from the selector we import from antrea is not good

Reply:
added metadat.namespace for field-selector